_build/data/transport.core.system_settings.php

```diff
@@ -902,6 +902,61 @@
   'area' => 'mail',
   'editedon' => null,
 ], '', true, true);
+
+$settings['mail_dkim_selector'] = $xpdo->newObject(modSystemSetting::class);
+$settings['mail_dkim_selector']->fromArray([
+  'key' => 'mail_dkim_selector',
+  'value' => '',
+  'xtype' => 'textfield',
+  'namespace' => 'core',
+  'area' => 'mail',
+  'editedon' => null,
+], '', true, true);
+$settings['mail_dkim_identity'] = $xpdo->newObject(modSystemSetting::class);
+$settings['mail_dkim_identity']->fromArray([
+  'key' => 'mail_dkim_identity',
+  'value' => '',
+  'xtype' => 'textfield',
+  'namespace' => 'core',
+  'area' => 'mail',
+  'editedon' => null,
+], '', true, true);
+$settings['mail_dkim_domain'] = $xpdo->newObject(modSystemSetting::class);
+$settings['mail_dkim_domain']->fromArray([
+  'key' => 'mail_dkim_domain',
+  'value' => '',
+  'xtype' => 'textfield',
+  'namespace' => 'core',
+  'area' => 'mail',
+  'editedon' => null,
+], '', true, true);
+$settings['mail_dkim_privatekeyfile'] = $xpdo->newObject(modSystemSetting::class);
+$settings['mail_dkim_privatekeyfile']->fromArray([
+  'key' => 'mail_dkim_privatekeyfile',
+  'value' => '',
+  'xtype' => 'textfield',
+  'namespace' => 'core',
+  'area' => 'mail',
+  'editedon' => null,
+], '', true, true);
+$settings['mail_dkim_privatekeystring'] = $xpdo->newObject(modSystemSetting::class);
+$settings['mail_dkim_privatekeystring']->fromArray([
+  'key' => 'mail_dkim_privatekeystring',
+  'value' => '',
+  'xtype' => 'textfield',
+  'namespace' => 'core',
+  'area' => 'mail',
+  'editedon' => null,
+], '', true, true);
+$settings['mail_dkim_passphrase'] = $xpdo->newObject(modSystemSetting::class);
+$settings['mail_dkim_passphrase']->fromArray([
+  'key' => 'mail_dkim_passphrase',
+  'value' => '',
+  'xtype' => 'textfield',
```

Comment:
```suggestion
  'xtype' => 'text-password',
```

I suggest converting the input field into a password field.